Trappist/src/Promact.Trappist.Core/Controllers/TestDashBoardController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Promact.Trappist.Repository.TestDashBoard;
+
+namespace Promact.Trappist.Core.Controllers
+{
+    [Route("api/TestDashboard")]
+    public class TestDashBoardController : Controller
+    {
+      
+     
+            public ITestDashBoardRepository _testDisplay;
```

Comment:
variable name should be _testDashboardRepository. Please provide variable name properly. this name _testDispay doesn't make any sense